### PR TITLE
docs(2.3.x) improve Admin API loopback instructions

### PR DESCRIPTION
**Pull request by kikito**
### Review

@reviewer

### Summary
This change fixes several problems in the Admin API loopback
instructions.

* The example uses Routes and Services instead of the deprecated Apis
  examples
* It differentiates more clearly which operations need to happen in
  127.0.0.1 in contrast to the ones which can be applied outside.
* It explains how to set up dbless kong using Docker and a declarative
  configuration
* It explains how to set up a PostgreSQL kong using Docker and a
  start configuration

### Reason

https://kongstrong.slack.com/archives/CDSTDSG9J/p1614186873017300

### CODE REVIEW

**Comment by kikito** [comment]
Once this gets a round of reviews, we need to replicate this change in previous versions of Kong (2.2, 2.1 ...). Keeping this in Draft status until then.

File: app/gateway-oss/2.3.x/secure-admin-api.md [hunk 1]
```diff
@@ -110,6 +110,112 @@ From here, simply apply desired Kong-specific security controls (such as
...
+    -e "KONG_PROXY_ACCESS_LOG=/dev/stdout" \
+    -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout" \
+    -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
+    -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
+    -e "KONG_ADMIN_LISTEN=0.0.0.0:8001, 0.0.0.0:8444 ssl" \
+    -v $(pwd):/home/kong
+    -p 8000:8000 \
+    -p 8443:8443 \
+    -p 8001:8001 \
+    -p 8444:8444 \
```
Comment (hutchic) [inline app/gateway-oss/2.3.x/secure-admin-api.md:151]: leave the admin api ports off

**Comment by kikito** [comment]
I have updated this PR to include all the 2.x series (EE and CE). The only difference between EE and CE is that the former uses the `kong-ee` docker image while CE uses the `kong` image.

Please give this a final review before merging. Are there any Kong versions missing? Should the instructions be a little different for older versions in 2.x?

File: app/enterprise/2.1.x/secure-admin-api.md [hunk 2]
```diff
@@ -115,6 +119,105 @@ From here, simply apply desired Kong-specific security controls (such as
...
+  plugins:
+  - name: key-auth
+
+consumers:
+- username: admin
+  keyauth_credentials:
+  - key: secret
+```
+
+Under this configuration, the Admin API will be available via the `/admin-api` but only for requests accopanied with `?apikey=secret` query
```
Comment (tlblessing) [inline app/enterprise/2.1.x/secure-admin-api.md:142]:
```suggestion
Under this configuration, the Admin API will be available through the `/admin-api`, but only for requests accompanied with `?apikey=secret` query
```

avoid Latin phrases and abbreviations per style guide

File: app/enterprise/2.2.x/secure-admin-api.md [hunk 3]
```diff
@@ -115,6 +119,105 @@ From here, simply apply desired Kong-specific security controls (such as
 [IP restrictions][ip-restriction], or [access control lists][acl]) as you would
 normally to any other Kong API.
 
+If you are using docker to host Kong, you can accomplish a similar task via a declarative configuration such as this one:
```
Comment (tlblessing) [inline app/enterprise/2.2.x/secure-admin-api.md:122]: Same comments as above

File: app/enterprise/2.1.x/secure-admin-api.md [hunk 4]
```diff
@@ -115,6 +119,105 @@ From here, simply apply desired Kong-specific security controls (such as
...
+parameters.
+
+Assuming that the file above is stored in `$(pwd)/kong.yml`, a dbless Kong can use it as it starts like this:
```
Comment (tlblessing) [inline app/enterprise/2.1.x/secure-admin-api.md:145]:
```suggestion
Assuming that the file above is stored in `$(pwd)/kong.yml`, a DB-less {{site.ee_product_name}} can use it as it starts like this:
```

For consistency https://docs.konghq.com/hub/plugins/compatibility/. I know just saying "Kong" was convenient but we should follow proper use of variables.

File: app/enterprise/2.3.x/secure-admin-api.md [hunk 5]
```diff
@@ -115,6 +119,105 @@ From here, simply apply desired Kong-specific security controls (such as
 [IP restrictions][ip-restriction], or [access control lists][acl]) as you would
 normally to any other Kong API.
 
+If you are using docker to host Kong, you can accomplish a similar task via a declarative configuration such as this one:
```
Comment (tlblessing) [inline app/enterprise/2.3.x/secure-admin-api.md:122]: Same comments as above

**Review by tlblessing — CHANGES_REQUESTED** [review]
Thank you for the PR @kikito. Some suggested edits.

File: app/enterprise/2.1.x/secure-admin-api.md (hunk 2 above)
Comment (kikito) [reply app/enterprise/2.1.x/secure-admin-api.md]: Ah, yes I have a tendency to use those. I suspect I have some Ancient Roman ancestry.

File: app/enterprise/2.1.x/secure-admin-api.md (hunk 4 above)
Comment (kikito) [reply app/enterprise/2.1.x/secure-admin-api.md]: I didn't know we were using those environment variables now. I have changed them (`site.ee_product_name` in Enterprise pages and `site.ce_product_name` in Community pages)

File: app/enterprise/2.3.x/secure-admin-api.md (hunk 5 above)
Comment (kikito) [reply app/enterprise/2.3.x/secure-admin-api.md]: Done!

File: app/enterprise/2.2.x/secure-admin-api.md (hunk 3 above)
Comment (kikito) [reply app/enterprise/2.2.x/secure-admin-api.md]: Done!

**Comment by kikito** [comment]
@tlblessing thank you for the review, I implemented all the changes.

**Review by tlblessing — APPROVED** [review]
Thanks @kikito !

**Pull request merged**